src/_helpers/typography.ts

```diff
@@ -207,7 +207,7 @@ const subheading = css`
 
 const subheadingLarge = css`
   font-family: ${fontFamily.sans};
-  font-size: ${fontSize[14]};
```

Comment:
We don't have 13px in our typography scale which is why I've been avoiding this.

Reply:
I would have left it to 14px as well. Not sure why we are coming back to 13px. This also create a disconnection between Figma and Tetra. @domyen we should avoid this going forward.